pkg/blobstore/configuration/new_blob_access.go

```diff
@@ -43,6 +44,7 @@ func newRedisClient(opt *redis.Options) *redis.Client {
 }
 
 func newNestedBlobAccessBare(configuration *pb.BlobAccessConfiguration, creator BlobAccessCreator) (BlobAccessInfo, string, error) {
+	lifeCycle := global.NewLifeCycleManager(0)
```

Comment:
Shall I put the `lifeCycleManager` inside the `BlobAccessCreator` or as a separate argument?